python_ta/typecheck/base.py

```diff
@@ -263,7 +265,13 @@ def can_unify(self, t1, t2):
                         return False
                 return True
             else:
-                False
+                return False
+        elif isinstance(t1, GenericMeta):
```

Comment:
I'm not sure why these are necessary. Won't they be caught by the `t1 != t2` below?